2022 - Mechanicum.cat

```diff
@@ -1830,6 +1830,7 @@
     <selectionEntry id="0eae-0c52-1087-a3b0" name="Tech-Priest Auxilia" publicationId="bde1-6db1-163b-3b76" page="28" hidden="false" collective="false" import="true" type="unit">
       <categoryLinks>
         <categoryLink id="0f49-7308-c6f1-e967" name="Line Sub-type:" hidden="false" targetId="6399-5c65-7833-1025" primary="false"/>
+        <categoryLink targetId="86dd-51d8-3669-68ca" id="2b8b-9a5c-efda-9e8b" primary="false" name="Tech-Priest Auxilia"/>
```

Comment:
No idea if these render anywhere but do we want to consider setting this to hidden?